README.md

```diff
@@ -1,48 +1,74 @@
-repo-openio-sds
-===============
+[![Build Status](https://travis-ci.org/open-io/ansible-role-openio-repository.svg?branch=master)](https://travis-ci.org/open-io/ansible-role-openio-repository)
+# Ansible role `repository`
 
-Ansible role that configure OpenIO SDS repositories.
+An Ansible role for setup OpenIO's repositories. Specifically, the responsibilities of this role are to:
 
-Requirements
-------------
+- Setup key and repository for an OpenIO product and a specific distribution
+- Manage credentials for private repositories
 
-Only supports RedHat 7 and Ubuntu 16.04.
+## Requirements
 
-Role Variables
---------------
+- Ansible 2.4+
 
-Available variables are listed below, along with default values (see defaults/main.yml):
+## Role Variables
 
-    openio_sds_release: '16.04'
-OpenIO SDS release installed. Defaults to the latest stable release.
 
-    openio_sds_repo_baseurl: http://mirror.openio.io/pub/repo/openio
-OpenIO base mirror URL for repositories.
+| Variable   | Default | Comments (type)  |
+| :---       | :---    | :---             |
+| `openio_repository_creds` | `""` | Credentials string |
+| `openio_repository_disable_policy_autostart` | `true` | (Ubuntu) Disable the autostart of unconfigured packages |
+| `openio_repository_distribution` | `'17.04'` | Distribution of OpenIO |
+| `openio_repository_manage_openstack_repository` | `true` | Allow the installation of Openstack's repository |
+| `openio_repository_openstack_distribution` | `pike` | Openstack's distribution |
+| `openio_repository_pass` | `""` | Password for private repository |
```

Comment:
maybe "password" will be more explicit (I know it was my naming, but maybe it was a wrong choice after all)

Reply:
ok password is more precise :) Just like `distribution`